lib/logger_humio_backend.ex

```diff
@@ -207,38 +207,34 @@ defmodule Logger.Backend.Humio do
     opts = Keyword.merge(env, opts)
     Application.put_env(:logger, name, opts)
 
+
+    # do these fields need to be logged or not? to be or to not to be
     host = Keyword.get(opts, :host, "")
     token = token(Keyword.get(opts, :token, ""))
 
-    ingest_api = Keyword.get(opts, :ingest_api, @default_ingest_api)
-    client = Keyword.get(opts, :client, @default_client)
-    level = Keyword.get(opts, :level, @default_level)
-    metadata = Keyword.get(opts, :metadata, @default_metadata)
-    format = opts |> Keyword.get(:format, nil) |> Formatter.compile()
-    max_batch_size = Keyword.get(opts, :max_batch_size, @default_max_batch_size)
-    flush_interval_ms = Keyword.get(opts, :flush_interval_ms, @default_flush_interval_ms)
-    debug_io_device = Keyword.get(opts, :debug_io_device, @default_debug_io_device)
-    iso8601_format_fun = TimeFormat.iso8601_format_fun()
-    fields = Keyword.get(opts, :fields, %{})
-    tags = Keyword.get(opts, :tags, %{})
+    # false by default
+    print_config? = Keyword.get(opts, :print_config?, false)
+
+    keyword_list = [
```

Comment:
```suggestion
    config = [
```

Naming variables after their type is generally a bad idea; variable names should always describe their contents.